podcasts/SwiftUI/HorizontalCarouselCard.swift

```diff
@@ -0,0 +1,93 @@
+import SwiftUI
+
+struct HorizontalCarouselCard: View {
+    let item: any HorizontalCarouselItemRepresentable
+
+    private var isiPad: Bool {
```

Comment:
Instead of checking for iPad did you consider checking for size class?
Like this:
`@Environment(\.horizontalSizeClass) var horizontalSizeClass`

This could be relevant for iPad multitask modes